src/System.Linq.Expressions/ref/System.Linq.Expressions.csproj

```diff
@@ -8,6 +8,8 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netcoreapp-Release|AnyCPU'" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'uap-Debug|AnyCPU'" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'uap-Release|AnyCPU'" />
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'uapaot-Debug|AnyCPU'" />
```

Comment:
These guys are not needed for two reasons:
 - We don't need a special build for uapaot since we can just use the same build that is produced for uap. We only do uapaot specific configurations when we actually have an implementation difference, which is never the case for refs.
 - IF we did want a new uapaot specific build, the right way to add the configuration is to add uapaot in the configurations.props and then just call `msbuild /t:UpdateVSConfigurations` from the root of corefx to update all of these conditions plus solution files.